report.tex

```diff
@@ -1,8 +1,17 @@
 \documentclass{scrreprt}	%KOMA-Script Klasse für report; 
 
 % HEADER
-\usepackage[english, german]{babel}	% Paket für Sprachselektion, in diesem Fall für deutsches Datum etc
+\usepackage[english, ngerman]{babel}	% Paket für Sprachselektion, in diesem Fall für deutsches Datum etc
 \usepackage[utf8]{inputenc}	% Paket für Umlaute; verwende utf8 Kodierung in TexWorks 
+\usepackage[T1]{fontenc} %ö,ü,ä werden richtig kodiert
```

Comment:
Why is this needed? The package above should do the trick.

Reply:
http://texwelt.de/wissen/fragen/5537/was-macht-eigentlich-usepackaget1fontenc

I think the main difference is the following: For example if you search in the pdf for  "können" you won't find it without this package.

Reply:
Do we then still need the package above?

Reply:
Absolutly! Just try it out :)